packager/media/codecs/aac_audio_specific_config.cc

```diff
@@ -162,6 +162,33 @@ bool AACAudioSpecificConfig::ConvertToADTS(std::vector<uint8_t>* buffer) const {
   return true;
 }
 
+std::vector<uint8_t> AACAudioSpecificConfig::GetAACwithADTSHeader(const uint8_t* data, size_t length) const {
```

Comment:
Can you update ConvertToADTS function directly?

You may change the function signature to:

bool ConvertToADTS(const uint8_t* data, size_t data_size, std::vector<uint8_t>* output_buffer)